permission_handler_android/android/src/main/java/com/baseflow/permissionhandler/PermissionConstants.java

```diff
@@ -28,28 +28,29 @@ final class PermissionConstants {
     static final int PERMISSION_GROUP_PHOTOS_ADD_ONLY = 10;
     static final int PERMISSION_GROUP_REMINDERS = 11;
     static final int PERMISSION_GROUP_SENSORS = 12;
-    static final int PERMISSION_GROUP_SMS = 13;
-    static final int PERMISSION_GROUP_SPEECH = 14;
-    static final int PERMISSION_GROUP_STORAGE = 15;
-    static final int PERMISSION_GROUP_IGNORE_BATTERY_OPTIMIZATIONS = 16;
-    static final int PERMISSION_GROUP_NOTIFICATION = 17;
-    static final int PERMISSION_GROUP_ACCESS_MEDIA_LOCATION = 18;
-    static final int PERMISSION_GROUP_ACTIVITY_RECOGNITION = 19;
-    static final int PERMISSION_GROUP_UNKNOWN = 20;
-    static final int PERMISSION_GROUP_BLUETOOTH = 21;
-    static final int PERMISSION_GROUP_MANAGE_EXTERNAL_STORAGE = 22;
-    static final int PERMISSION_GROUP_SYSTEM_ALERT_WINDOW = 23;
-    static final int PERMISSION_GROUP_REQUEST_INSTALL_PACKAGES = 24;
-    static final int PERMISSION_GROUP_APP_TRACK_TRANSPARENCY = 25;
-    static final int PERMISSION_GROUP_CRITICAL_ALERTS = 26;
-    static final int PERMISSION_GROUP_ACCESS_NOTIFICATION_POLICY = 27;
-    static final int PERMISSION_GROUP_BLUETOOTH_SCAN = 28;
-    static final int PERMISSION_GROUP_BLUETOOTH_ADVERTISE = 29;
-    static final int PERMISSION_GROUP_BLUETOOTH_CONNECT = 30;
-    static final int PERMISSION_GROUP_NEARBY_WIFI_DEVICES = 31;
-    static final int PERMISSION_GROUP_VIDEOS = 32;
-    static final int PERMISSION_GROUP_AUDIO = 33;
-    static final int PERMISSION_GROUP_SCHEDULE_EXACT_ALARM = 34;
+    static final int PERMISSION_GROUP_SENSORS_ALWAYS = 13;
```

Comment:
Please add the new permission at the bottom of the constant list, by renumbering the permissions we create a large breaking change since the contract between the native and Dart side is public. 

So please revert the renumbering and simply create a new entry at the bottom of the constant list for this new permission.